pkg/cmd/container/list.go

```diff
@@ -107,6 +107,7 @@ func (x *ListItem) Label(s string) string {
 
 func prepareContainers(ctx context.Context, client *containerd.Client, containers []containerd.Container, options types.ContainerListOptions) ([]ListItem, error) {
 	listItems := make([]ListItem, len(containers))
+	snapshottersCache := map[string]containerdutil.Snapshotter{}
```

Comment:
I think it's still better to use containerd's `snapshots.Snapshotter` here, which is more common/well-known than `containerdutil.Snapshotter`.

Reply:
Well, the point of this PR is specifically to solve performance issues with `ps` and `images`.
This method (`List`) is used solely by `ps` and is at the core of the issue.

Do you have specific concerns about `containerdutil.Snapshotter` that I could address?

Thanks for your review @djdongjin

Reply:
I mean `containerdutil.Snapshotter` is just an alias to `snapshots.Snapshotter` from containerd right? Personally I prefer to use the original containerd interface directly instead of alias.

When aliases are used in other packages (e.g., here). It's not clear if `containerdutil.Snapshotter` is some nerdctl-customized interface or it's the same as containerd `snapshots.Snapshotter`. Also leave the comment somewhere else.

Reply:
Oh I see your point now.
Misread your initial comment. Thanks for clarifying.
And point taken in the other comment.
I am fine removing these aliases and using the containerd interfaces instead.